theme/home.html

```diff
@@ -43,28 +43,27 @@ <h1>{{ homepage.hero.header }}</h1>
               {{ button.name }}
             </a>
           {% endfor %}
+          {% if config.theme.language == "en" %}
+            <a href="https://discuss.privacyguides.net/" title="Join the Privacy Guides forum" class="md-button">
+              Ask A Question
```

Comment:
I like showcasing a link to the Discourse forum (it is one of the strengths of Privacy Guides after all), but I'm not sure about the button text.

Ideally, people who land on the site for the first time would browse the Knowledge Base or Recommendations before asking a question on the forum, rather than ask a question on the forum that is already answered on the site (here is a [recent example](https://discuss.privacyguides.net/t/encrypted-storage/18910) of this). This is just my opinion though; I would love other people's take on this.
```suggestion
              Join Our Community
```

Reply:
> It's more of a low barrier than having to sign up for something.

This was also my thought process, so I don't think I like @redoomed1's suggested change here, but I'm open to other suggestions that aren't just saying "join us" too, because ask a question isn't _great_ either.

Reply:
Maybe something along the lines of "Participate in [...]"?

Reply:
Upon further reflection, I think we should just not add this button at all actually.